packages/legend-application-studio/src/components/editor/side-bar/Explorer.tsx

```diff
@@ -746,6 +751,16 @@ const ProjectExplorerActionPanel = observer((props: { disabled: boolean }) => {
           <FileImportIcon />
         </button>
       )}
+      {!editorStore.isInViewerMode && (
```

Comment:
it should be enabled in viewer mode as well. We have readOnly checks to disable editing but users should be able to view the config editor